src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -939,23 +954,33 @@ void TwitchMessageBuilder::parseHighlights()
         {
             this->highlightVisual_ = true;
             this->message().flags.set(MessageFlag::Highlighted);
+            this->message().highlightColor = userHighlight.getColor();
         }
 
-        if (userHighlight.getAlert())
+        if (userHighlight.hasAlert())
         {
             this->highlightAlert_ = true;
         }
 
-        if (userHighlight.getSound())
+        if (userHighlight.hasSound())
         {
             this->highlightSound_ = true;
+            // Use custom sound if set, otherwise use the fallback sound
+            if (!userHighlight.getSoundUrl().toString().isEmpty())
```

Comment:
Replace with a `HighlightPhrase::hasCustomSound` that does the same thing

Reply:
Resolved in d938649d.